CMakeLists.txt

```diff
@@ -13,20 +13,33 @@
 # You should have received a copy of the GNU Lesser General Public License along
 # with the SymCC runtime. If not, see <https://www.gnu.org/licenses/>.
 
-cmake_minimum_required(VERSION 3.5)
-project(SymRuntime)
+cmake_minimum_required(VERSION 3.22)
```

Comment:
I agree that we can increase the minimum required version, but I wonder if this too recent :sweat_smile: Ubuntu 20.04 LTS seems to ship with CMake 3.16, and the discussion in https://github.com/eurecom-s3/symcc/issues/130 suggests that we may still want to support it. I think FuzzBench hasn't moved away from 20.04 yet.